HandEyeCalibration/UtilizeEyeInHandCalibration/UtilizeEyeInHandCalibration.cpp

```diff
@@ -0,0 +1,83 @@
+/*
+Utilize the result of eye-in-hand calibration to transform (picking) point
+coordinates from the camera frame to the robot base frame.
+*/
+
+#include <Eigen/Core>
```

Comment:
It's customary to sort includes like this: own project, third party, standard library. I suggest doing:
```c++
#include <Eigen/Core>
#include <opencv2/core/core.hpp>

#include <cmath>
#include <iostream>
```

Reply:
It's possible to automate this btw, we've done so in the main repository. We just have to prioritize doing it for this repo too.